### PR TITLE
Adverts

### PR DESCRIPTION
**Pull request by Lordisch**

### CODE REVIEW

File: src/components/SettingComponents/Adverts.vue [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+export default {
+  name: 'Adverts',
+  components: {
+    DialogForm,
+    SettingTitle,
+    DeleteConfirmationDialog,
+  },
+  data() {
+    return {
+      advert: [],
```
Comment (rFranzr) [inline src/components/SettingComponents/Adverts.vue:119]: advert: [] is not used

**Review by rFranzr — COMMENTED** [review]
Code looks good, w8 for review of @ibot3

File: src/components/HomeComponents/ServerStatus.vue [hunk 2]
```diff
@@ -25,6 +25,9 @@
                     <span v-if="server.users_current != null">
                     {{ server.users_current }}
                   </span>
+                    <span v-else-if="server.status === 'OFFLINE'">
+                      0
+                    </span>
```
Comment (ibot3) [inline src/components/HomeComponents/ServerStatus.vue:30]: Not related to the PR, put it on the develop branch or a separate branch next time.

**Review by ibot3 — APPROVED** [review]
Looks good, but I don't see a commit change of the `lang` submodule yet.

**Pull request merged**